test/ui-testing/wysiwyg.test.js

```diff
@@ -0,0 +1,53 @@
+const timeout = process.env.SLOWMO ? 60000 : 10000;
```

Comment:
We can try increasing this to above 10 seconds for all tests...?

Reply:
Sure we could do that. But may I ask please, why do you think we should increase the timeout? Did you experience any timeout recently?
@jywarren

Reply:
Based on the message in the logs:

`    : Timeout - Async callback was not invoked within the 10000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 10000 ms timeout specified by jest.setTimeout.Error:`

I am guessing that for some reason it took longer. It's worth a try!

Reply:
And that was the same as with one of the earlier bold test failures!

Reply:
Oh ok, I get. I think I got that error sometimes and when I ran the test again it works fine. But it's ok, we could increase the timeout value to say 15000ms or 20000ms. 
Is that much or enough, what do you think?